model/src/wav_import_export.F90

```diff
@@ -357,6 +360,11 @@ subroutine import_fields( gcomp, time0, timen, rc )
       if (state_fldchk(importState, 'So_u')) then
         call SetGlobalInput(importState, 'So_u', vm, global_data, rc)
         if (ChkErr(rc,__LINE__,u_FILE_u)) return
+        do isea = 1,nsea 
+        if(casename == "ufs.hafs" .and. abs(global_data(isea)-fillv).lt.0.01) then
```

Comment:
What about: 

```
if(casename == "ufs.hafs") then 
  do isea = 1,nsea
      if abs(global_data(isea)-fillv).lt.0.01) then 
          global_data(isea)=0.0


```

That way for non-hafs applications you are not going through this loop.

Reply:
@JessicaMeixner-NOAA I will revise the wav_import_export.F90 file.
The changes in WW3 alone will not change any baselines in the UFS regression tests since ocean-current fields are not transferred to WW3 using the emc/develop branch of CMEPS. A related CMEPS PR will transfer the fields.

Reply:
Thanks for the explanation @binli2337 !

Reply:
@binli2337 In the global model, we do import ``So_u`` and ``So_v`` to WAV.

Reply:
@JessicaMeixner-NOAA The ufs-weather-model branch is at
https://github.com/binli2337/ufs-weather-model/tree/202501b/ocean_wave_update. A PR will be submitted soon.